src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
```

Comment:
Consider creating here the set of constants and helper functions to use instead of magic strings and numbers and minimizing code repeating

Reply:
For example: 
```const MAN = 'm'; ```